### DISCUSSION TITLE
Slideshow looks so washed-up on Windows (10 64bit)

### DISCUSSION THREAD

**Discussion by theJazzman42** [KejPi/AbracaDABra#82] [General]
On Linux it looks perfect, on Windows it's like "CGA graphics". Sort of...

![image](https://github.com/KejPi/AbracaDABra/assets/123947155/31049829-0651-40ca-8186-81d7bf367000)

**Comment by KejPi** [comment]
I am afraid I cannot do anything with that - it is how Qt or Windows does it, there is no difference in my SLS implementation between Linux and Win. I would suggest you to stay on the original picture size (resize the application window) not the scaled version as you show in the screenshot. If you resize application to its minimum size, is is still bad?

**Reply by theJazzman42** [reply]
Hmm... Already minimized the window as much as possible...

**Reply by KejPi** [reply]
I can clearly see black stripes on left and right it means application window is not on its minimum size.

**Comment by KejPi** [comment]
This is how it looks like on my Win10
![screenshot](https://github.com/KejPi/AbracaDABra/assets/6438380/0087fa4d-228e-4031-9615-20005e88216a)

**Reply by theJazzman42** [reply]
It's because I made a screenshot of the entire desktop and cut the window not properly enough.

**Comment by KejPi** [comment]
No, it is not, there are black stripes on the sides of the slide. If that slide is 320x240 as typically is, there are no black stripes when application windows is in its minimum size. In such case the slide is as good as the broadcaster sends it. In your case it seems that the slide could be scaled up a bit.

**Comment by theJazzman42** [comment]
![image](https://github.com/KejPi/AbracaDABra/assets/123947155/e5e13b45-7f8c-45c6-9b90-300841bd725c)

**Comment by KejPi** [comment]
I agree it looks bit blurry. The question is whether it is because the quality of source data is low or if Windows displays it in bad quality. From the screenshot I would say that you have HiDPI screen. Windows always has some graphical problems on HiDPI - small fonts or big fonts or icons etc. You can try some hints from this link: https://doc.qt.io/qt-6/highdpi.html
By the way, have you tried to record the stream and play it from Linux for comparison?

**Comment by andimik** [comment]
Which zoom factor do you have? Try 100%.

**Comment by theJazzman42** [comment]
I do. 

Before my Noxon stick went kaput I used the DAB Player written by Andreas Gsinn. Didn't have this problem.

**100 % zoom**. 

![image](https://github.com/KejPi/AbracaDABra/assets/123947155/641d7d28-ea8e-4927-b9e1-7252854e1e76)

**Comment by KejPi** [comment]
For comparison, this is how the SLS looks like on HiDPI screen on Win and on Linux running in VBox on the same machine in comparison with DAB Player. Actually the picture from DAB player shows potential reason of blurry image. While DAB player shows SLS in physical pixels (very small image on HiDPI), in AbracaDABra the minimum size is limited by other widgets in UI (it could be also because Qt somehow considers DPI). In any case I cannot change it. 


![screenshotWin](https://github.com/KejPi/AbracaDABra/assets/6438380/b8703f09-18dc-4a28-994e-ff0199356535)

![screenshotDABPlayer](https://github.com/KejPi/AbracaDABra/assets/6438380/e6d9408e-3389-4823-9b46-2b602055323c)

![Screenshot from 2023-06-12 21-08-04](https://github.com/KejPi/AbracaDABra/assets/6438380/42a65e8a-77c2-4be6-8d44-7c40aa6c3245)
